Formula/curlcpp.rb

```diff
@@ -0,0 +1,25 @@
+class Curlcpp < Formula
+  desc "Object oriented C++ wrapper for CURL (libcurl)"
+  homepage "https://josephp91.github.io/curlcpp"
+  url "https://github.com/JosephP91/curlcpp/archive/refs/tags/1.5.tar.gz"
+  sha256 "d1c236be862965d5882a842be3112b16e7f49d6fdff9906f7aec71ee8669d60e"
+  license "MIT"
+
+  depends_on "cmake" => :build
+  depends_on "make" => :build
+  depends_on "curl"
+
+  def install
+    ENV.deparallelize
+    Dir.mkdir("build")
+    chdir "build" do
+      system "cmake", "..", *std_cmake_args
+      system "make"
+      system "make", "install"
+    end
+  end
+
+  test do
+    system "false"
```

Comment:
We need a test that exercises the some of the functionality of the app. Version checks or usage checks (foo --version or foo --help) are not sufficient, as explained in the formula cookbook.

In most cases, a good test would involve running a simple test case: run #{bin}/foo input.txt.

- Then you can check that the output is as expected (with assert_equal or assert_match on the output of shell_output)
- You can also check that an output file was created, if that is expected: assert_predicate testpath/"output.txt", :exist?

Some advice for specific cases:

- If the formula is a library, compile and run some simple code that links against it. It could be taken from upstream's documentation / source examples.
- If the formula is for a GUI program, try to find some function that runs as command-line only, like a format conversion, reading or displaying a config file, etc.
- If the software cannot function without credentials, a test could be to try to connect with invalid credentials (or without credentials) and confirm that it fails as expected.
- Same if the software requires a virtual machine, docker instance, etc. to be running.